notebooks/AllSupportedTypes.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
This notebook tests all compression types, right? I was confused by the name. 

Maybe call it `AllCompressions` and add a text cell with an explanation.